deploy/metrics-server-deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: metrics-server
+  namespace: kube-system
```

Comment:
This shouldn't go into `kube-system`.  We need to limit the number of pods created in this namespace since pods mount SA tokens and the SA tokens in `kube-system` are very powerful.  You should create your own namespace, probably `kube-metrics` to contain this data.